README.md

```diff
@@ -43,11 +30,80 @@ Please always double-check the version of Keptn you are using compared to the ve
 |   0.9.0 - 0.9.2  | keptncontrib/prometheus-service:0.7.0  |
 |   0.10.0         | keptncontrib/prometheus-service:0.7.1  |
 
-## Setup Prometheus Monitoring
+## Installation instructions
+
+### Setup Prometheus Monitoring
 
 Keptn does not install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
 
-Some environment variables have to set up in the prometheus-service deployment
+The easiest way would be to setup Prometheus using helm, e.g.:
+```console
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm install prometheus prometheus-community/prometheus --namespace monitoring
+```
+
+### Install prometheus-service
+
+Please replace the placeholders in the commands below. Examples are provided.
+
+* `<VERSION>`: prometheus-service version, e.g., `0.7.1`
+* `<PROMETHEUS_NS>`: If prometheus is installed in the same Kubernetes cluster, the namespace needs to be provided, e.g., `monitoring`
+* `<PROMETHEUS_ENDPOINT>`: Endpoint for prometheus (primarily used for fetching metrics), e.g., `http://prometheus-server.monitoring.svc.cluster.local:80`
+* `<ALERT_MANAGER_NS>`: if prometheus alert manage ris installed in the same Kubernetes cluster, the namespace needs to be provided, e.g., `monitoring`
```

Comment:
"alert manage ris" -> "alert manager is"

Reply:
fixed